ui/apps/everest/src/components/cluster-form/resources/constants.ts

```diff
@@ -162,6 +162,7 @@ export const SHARDING_DEFAULTS = {
 
 export const resourcesFormSchema = (passthrough?: boolean) => {
   const objectShape = {
+    [DbWizardFormFields.dbType]: z.string().optional(),
```

Comment:
why we need dbType in the schema? doesn't zod allow you to take data from other part of the form?